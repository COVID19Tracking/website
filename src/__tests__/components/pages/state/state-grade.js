import React from 'react'
import renderer from 'react-test-renderer'
import StateGrade from '../../../../components/pages/state/state-grade'

describe('Components : Pages : State : State grade', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<StateGrade letterGrade="a" />).toJSON()
    expect(tree).toMatchSnapshot()

    const aPlusTree = renderer.create(<StateGrade letterGrade="A+" />).toJSON()
    expect(aPlusTree).toMatchSnapshot()

    const naTree = renderer.create(<StateGrade />).toJSON()
    expect(naTree).toMatchSnapshot()
  })
})
