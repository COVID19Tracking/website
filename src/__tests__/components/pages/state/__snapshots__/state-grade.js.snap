// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Pages : State : State grade renders correctly 1`] = `
<p
  className="state-grade stateGrade"
>
  <span>
    Current
     
    <a
      href="/about-data#state-data-quality-grades"
    >
      data grade
    </a>
    :
  </span>
  <img
    alt="Grade a"
    src="test-file-stub"
    width="28px"
  />
</p>
`;

exports[`Components : Pages : State : State grade renders correctly 2`] = `
<p
  className="state-grade stateGrade"
>
  <span>
    Current
     
    <a
      href="/about-data#state-data-quality-grades"
    >
      data grade
    </a>
    :
  </span>
  <img
    alt="Grade A+"
    src="test-file-stub"
    width="28px"
  />
</p>
`;

exports[`Components : Pages : State : State grade renders correctly 3`] = `
<p
  className="state-grade stateGrade"
>
  <span>
    Current
     
    <a
      href="/about-data#state-data-quality-grades"
    >
      data grade
    </a>
    :
  </span>
  <img
    alt="Grade na"
    src="test-file-stub"
    width="28px"
  />
</p>
`;
